clients/client-accessanalyzer/protocols/Aws_restJson1.ts

```diff
@@ -1165,7 +1168,7 @@ export const deserializeAws_restJson1CreateAccessPreviewCommand = async (
   };
   const data: any = await parseBody(output.body, context);
   if (data.id !== undefined && data.id !== null) {
```

Comment:
Don't know how the generator is structured, but we should be able to remove this wrapping `if` function that appears a lot as well. (Though I wouldn't block on it)

Reply:
This is probably due to differences with handling of top-level members.